app/dao/services_dao.py

```diff
@@ -453,6 +453,34 @@ def fetch_aggregate_stats_by_date_range_for_all_services(start_date, end_date, i
     return query.all()
 
 
+@statsd(namespace='dao')
+def fetch_new_aggregate_stats_by_date_range_for_all_services(start_date, end_date):
+    start_date = get_london_midnight_in_utc(start_date)
+    end_date = get_london_midnight_in_utc(end_date + timedelta(days=1))
+    table = NotificationHistory
+
+    if start_date >= datetime.utcnow() - timedelta(days=7):
+        table = Notification
+
+    query = db.session.query(
```

Comment:
Can we use the new FT_NOTIFICATION_STATUS table for this?
We would need to do get "todays" data from notifications, but for other data it could be FT_NOTIFICATION_STATUS.
But it's ok to save that for another PR